browser/src/control/Control.Menubar.js

```diff
@@ -127,6 +127,7 @@ L.Control.Menubar = L.Control.extend({
 					{name: _('Toggle UI Mode'), id: 'toggleuimode', type: 'action'},
 					{name: _('Show Ruler'), id: 'showruler', type: 'action'},
 					{name: _('Show Status Bar'), id: 'showstatusbar', type: 'action'},
+					{name: _('Collapse Notebook Bar'), id: 'collapsenotebookbar', type: 'action'},
```

Comment:
For compact view we shouldn't add collapsenotebookbar menu item because we are now in the compact view.

But maybe we could instead add something like id `togglemenubar` and name `hide menu bar`  new menu item. In here we would then call `this._map.uiManager.toggleMenubar();` But let's make not do that now and make this PR easier

Reply:
I opened a new issue : #6995